src/main/java/dev/jbang/cli/Setup.java

```diff
@@ -1,170 +0,0 @@
-package dev.jbang.cli;
```

Comment:
remind me again why `setup` shouldn't stay as top level command ? since it also setup j! alias + PATH it seems more like related but independent from `app` ?

Reply:
Not sure what `j! alias + PATH` is but basically `app` does not make sense without `setup` and vice versa. The only thing `setup` does is make `app` work, it doesn't really do anything more than that. Yes, it seems it also sets up Jbang for zero-install but you could say that zero-install is like doing `jbang app install jbang`

Reply:
in setup you add "alias j!=jbang", thats what is deocupled from app....

so zero install is now:  iex "& { $(iwr -useb https://ps.jbang.dev) } app setup" ?

Reply:
The alias seems unrelated yes, but you can't run setup just to set up an alias. It will always do the setup for the PATH. (And `j!` is nice but it might be difficult to get working for PowerShell and will most likely never work for CMD. So we can't really use it in documentation etc)

And yes, that would the command be right now.

Reply:
powershell: `Set-Alias j! jbang`

cmd: `echo jbang > j!.cmd`

it actually works ;)

Reply:
PS1: and where do we put that command? :-)
Although I've heard there is some "startup folder" for PS scripts.
But we could do the same as for:

CMD: ok, so create a copy of the original script with a different name (or make it call the original script)? That could work I guess.